pom.xml

```diff
@@ -58,15 +58,15 @@
 
     <!-- Testing -->
     <ver.junit>4.13.2</ver.junit>  
-    <ver.mockito>5.6.0</ver.mockito>
+    <ver.mockito>5.7.0</ver.mockito>
     <ver.awaitility>4.2.0</ver.awaitility>
     <ver.contract.tests>0.2.0</ver.contract.tests>
 
     <!-- Dependencies -->
     <ver.slf4j>1.7.36</ver.slf4j>
     <ver.log4j2>2.21.0</ver.log4j2>
 
-    <ver.lucene>9.7.0</ver.lucene>
+    <ver.lucene>9.8.0</ver.lucene>
```

Comment:
(this is in the ANN)

Lucene 9.8.0 does not work with all Java21, possible some EAs, but e.g, Ubuntu shipped EA in the latest release due to the timing issues.

Reply:
Backed out to `9.7.0`